HouseRules_Essentials/Rules/PieceUseWhenKilledOverriddenRule.cs

```diff
@@ -12,7 +12,7 @@ public sealed class PieceUseWhenKilledOverriddenRule : Rule, IConfigWritable<Dic
     {
         public override string Description => "Piece UseWhenKilled lists are overridden";
 
-        protected override SpecialSyncData ModifiedData => SpecialSyncData.PieceData;
+        protected override SyncableTrigger ModifiedSyncables => SyncableTrigger.NewPieceModified | SyncableTrigger.StatusEffectDataModified;
```

Comment:
Also added `SyncableTrigger.StatusEffectDataModified` to the mix, as this rule could be used to trigger an ability like `Freeze`, that will cause some update to status effects.